device.go

```diff
@@ -174,9 +174,17 @@ func getDeviceName(s *sandbox, devID string) (string, error) {
 }
 
 func getPCIDeviceNameImpl(s *sandbox, pciID string) (string, error) {
-	pciAddr, err := getDevicePCIAddress(pciID)
-	if err != nil {
-		return "", err
+	var pciAddr string
+	var err error
+
+	// Take the input pciID as pciAddr if its format is standard BDF
+	if len(pciID) == 12 && len(strings.Split(pciID, ":")) == 3 {
```

Comment:
A b/d/f string contains only 2 `:`, as it contains 1 `.` to separate the `device` from the `function`.

Reply:
please don't use magic numbers (12, 3), define new constants and document them

Reply:
That's right. With 2 `:`, the b/d/f string is divided into 3 parts, which is what the `if` statement is looking for.